tanner/emulators/template_injection.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+import logging
+
+from urllib.parse import unquote
+from tanner.utils import patterns
+from tanner.config import TannerConfig
+from tanner.utils.aiodocker_helper import AIODockerHelper
+
+
+class TemplateInjection:
+
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.template_injection')
+        self.docker_helper = AIODockerHelper()
+
+    async def get_injection_result(self, payload):
+        execute_result = None
+        container_name = 'template_injection'
+        github_remote_path = TannerConfig.get('REMOTE_DOCKERFILE', 'GITHUB')
+
+        # Build the custom image
+        await self.docker_helper.setup_host_image(
+            remote_path=github_remote_path, tag='template_injection:latest')
+
+        if patterns.TEMPLATE_INJECTION_TORNADO.match(payload):
+
+            with open('tanner/files/engines/tornado.py', 'r') as f:
+                tornado_template = f.read() % payload
+
+            cmd = ["python3", "-c", tornado_template]
```

Comment:
Maybe you can just write `python` and install only python3 in dockerfile so that while upgrading multiple changes are not required.

Reply:
Good idea! made the changes in the emulator and dockerfile already installs `python3` :)